sklearn/preprocessing/_data.py

```diff
@@ -917,6 +917,9 @@ def inverse_transform(self, X, copy=None):
         check_is_fitted(self)
 
         copy = copy if copy is not None else self.copy
+        X = check_array(X, accept_sparse='csr', copy=self.copy, 
```

Comment:
```suggestion
        X = check_array(X, accept_sparse='csr', copy=copy, 
```

Some of the code below this that handles the csr conversion and the copying are not needed anymore.

Reply:
You can delete the lines
```python
if not sparse.isspmatrix_csr(X):
    X = X.tocsr()
    copy = False
if copy:
    X = X.copy()
```
and
```python
X = np.asarray(X)
if copy:
    X = X.copy()
```